layouts/blog/single.html

```diff
@@ -146,5 +146,5 @@ <h2 class="font-bold text-2xl sm:text-4xl flex">
 
 {{ define "appcode" }}
 <script src="{{ `js/blogSingle.js` | absURL }}"></script>
-<script src="{{ `js/newsletterForm.js` | absURL }}"></script>
+<script type="module" src="{{ `js/newsletterForm.js` | absURL }}"></script>
```

Comment:
Updating the type to resolve an error caused by importing a function in the script file.